### ISSUE TITLE
Error building

### ISSUE THREAD

**Issue by SteveEdson** [connors/photon#9]
First of all, this looks awesome. Cannot wait to use it.

I've followed the example, but I'm getting an error:

```
$ npm install
-
> electron-prebuilt@0.33.7 postinstall /Users/Steve/Downloads/photon-0.1-alpha/node_modules/electron-prebuilt
> node install.js

Downloading electron-v0.33.7-darwin-x64.zip
[============================================>] 100.0% of 37.93 MB (1.38 MB/s)
grunt-contrib-clean@0.6.0 node_modules/grunt-contrib-clean
└── rimraf@2.2.8

grunt-contrib-copy@0.8.1 node_modules/grunt-contrib-copy
├── file-sync-cmp@0.1.1
└── chalk@1.1.1 (escape-string-regexp@1.0.3, ansi-styles@2.1.0, supports-color@2.0.0, strip-ansi@3.0.0, has-ansi@2.0.0)

grunt-banner@0.5.0 node_modules/grunt-banner
└── chalk@1.1.1 (supports-color@2.0.0, ansi-styles@2.1.0, escape-string-regexp@1.0.3, strip-ansi@3.0.0, has-ansi@2.0.0)

grunt-contrib-sass@0.9.2 node_modules/grunt-contrib-sass
├── async@0.9.2
├── dargs@4.0.1 (number-is-nan@1.0.0)
├── cross-spawn@0.2.9 (lru-cache@2.7.0)
├── chalk@0.5.1 (ansi-styles@1.1.0, escape-string-regexp@1.0.3, supports-color@0.2.0, strip-ansi@0.3.0, has-ansi@0.1.0)
└── which@1.2.0 (is-absolute@0.1.7)

grunt-jekyll@0.4.2 node_modules/grunt-jekyll
└── tmp@0.0.28 (os-tmpdir@1.0.1)

grunt-contrib-concat@0.5.1 node_modules/grunt-contrib-concat
├── chalk@0.5.1 (ansi-styles@1.1.0, escape-string-regexp@1.0.3, supports-color@0.2.0, has-ansi@0.1.0, strip-ansi@0.3.0)
└── source-map@0.3.0 (amdefine@1.0.0)

load-grunt-tasks@3.2.0 node_modules/load-grunt-tasks
├── multimatch@2.0.0 (array-differ@1.0.0, array-union@1.0.1, minimatch@2.0.10)
└── findup-sync@0.2.1 (glob@4.3.5)

grunt-contrib-watch@0.6.1 node_modules/grunt-contrib-watch
├── async@0.2.10
├── lodash@2.4.2
├── gaze@0.5.2 (globule@0.1.0)
└── tiny-lr-fork@0.0.5 (debug@0.7.4, qs@0.5.6, faye-websocket@0.4.4, noptify@0.0.3)

time-grunt@1.2.1 node_modules/time-grunt
├── number-is-nan@1.0.0
├── figures@1.4.0
├── date-time@1.0.0
├── hooker@0.2.3
├── text-table@0.2.0
├── chalk@1.1.1 (escape-string-regexp@1.0.3, supports-color@2.0.0, ansi-styles@2.1.0, has-ansi@2.0.0, strip-ansi@3.0.0)
└── pretty-ms@1.4.0 (get-stdin@4.0.1, is-finite@1.0.1, parse-ms@1.0.0, plur@1.0.0, meow@3.4.2)

grunt-contrib-cssmin@0.13.0 node_modules/grunt-contrib-cssmin
├── chalk@1.1.1 (escape-string-regexp@1.0.3, supports-color@2.0.0, ansi-styles@2.1.0, has-ansi@2.0.0, strip-ansi@3.0.0)
├── clean-css@3.3.10 (commander@2.8.1, source-map@0.4.4)
└── maxmin@1.1.0 (figures@1.4.0, gzip-size@1.0.0, pretty-bytes@1.0.4)

grunt@0.4.5 node_modules/grunt
├── eventemitter2@0.4.14
├── which@1.0.9
├── dateformat@1.0.2-1.2.3
├── colors@0.6.2
├── getobject@0.1.0
├── async@0.1.22
├── rimraf@2.2.8
├── lodash@0.9.2
├── hooker@0.2.3
├── grunt-legacy-util@0.2.0
├── exit@0.1.2
├── nopt@1.0.10 (abbrev@1.0.7)
├── coffee-script@1.3.3
├── iconv-lite@0.2.11
├── minimatch@0.2.14 (sigmund@1.0.1, lru-cache@2.7.0)
├── underscore.string@2.2.1
├── glob@3.1.21 (inherits@1.0.2, graceful-fs@1.2.3)
├── grunt-legacy-log@0.1.2 (grunt-legacy-log-utils@0.1.1, lodash@2.4.2, underscore.string@2.3.3)
├── findup-sync@0.1.3 (lodash@2.4.2, glob@3.2.11)
└── js-yaml@2.0.5 (esprima@1.0.4, argparse@0.1.16)

grunt-contrib-connect@0.11.2 node_modules/grunt-contrib-connect
├── opn@1.0.2
├── async@0.9.2
├── connect-livereload@0.5.3
├── portscanner@1.0.0 (async@0.1.15)
├── morgan@1.6.1 (on-headers@1.0.1, basic-auth@1.0.3, depd@1.0.1, on-finished@2.3.0, debug@2.2.0)
├── connect@3.4.0 (utils-merge@1.0.0, parseurl@1.3.0, debug@2.2.0, finalhandler@0.4.0)
├── serve-static@1.10.0 (escape-html@1.0.2, parseurl@1.3.0, send@0.13.0)
└── serve-index@1.7.2 (escape-html@1.0.2, parseurl@1.3.0, batch@0.5.2, http-errors@1.3.1, mime-types@2.1.7, debug@2.2.0, accepts@1.2.13)

grunt-html@5.0.1 node_modules/grunt-html
├── async@1.4.0
└── chalk@1.1.0 (escape-string-regexp@1.0.3, ansi-styles@2.1.0, supports-color@2.0.0, strip-ansi@3.0.0, has-ansi@2.0.0)

grunt-sed@0.2.0 node_modules/grunt-sed
├── replace@0.3.0 (colors@0.5.1, minimatch@0.2.14, nomnom@1.6.2)
└── lodash@3.7.0

grunt-contrib-compress@0.13.0 node_modules/grunt-contrib-compress
├── prettysize@0.0.3
├── chalk@0.5.1 (ansi-styles@1.1.0, escape-string-regexp@1.0.3, supports-color@0.2.0, has-ansi@0.1.0, strip-ansi@0.3.0)
└── archiver@0.13.1 (buffer-crc32@0.2.5, async@0.9.2, lazystream@0.1.0, lodash@2.4.2, readable-stream@1.0.33, glob@4.3.5, tar-stream@1.1.5, zip-stream@0.5.2)

electron-prebuilt@0.33.7 node_modules/electron-prebuilt
├── extract-zip@1.1.1 (debug@0.7.4, async@0.9.0, yauzl@2.3.1, mkdirp@0.5.0, concat-stream@1.5.0)
└── electron-download@1.2.0 (path-exists@1.0.0, home-path@1.0.1, minimist@1.2.0, debug@2.2.0, mkdirp@0.5.1, mv@2.1.1, nugget@1.5.4)




$ npm start

> photon@0.1.0 start /Users/Steve/Downloads/photon-0.1-alpha
> electron example-components


npm ERR! Darwin 15.0.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "start"
npm ERR! node v4.1.1
npm ERR! npm  v2.14.4
npm ERR! code ELIFECYCLE
npm ERR! photon@0.1.0 start: `electron example-components`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the photon@0.1.0 start script 'electron example-components'.
npm ERR! This is most likely a problem with the photon package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     electron example-components
npm ERR! You can get their info via:
npm ERR!     npm owner ls photon
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/Steve/Downloads/photon-0.1-alpha/npm-debug.log
```

Along with an electron pop up saying:

> The app provided is not a valid electron app, please read the docs on how to write one:
> https://github.com/atom/electron/tree/master/docs
>
> Error: Cannot find module '/Users/Steve/Downloads/photon-0.1-alpha/example-components'

I realise it might just be because I haven't set anything up yet, I just presumed a basic boilerplate would work from the start.

**Comment by ggamel** [comment]
This has been my experience as well. See below:

```
❯ npm start

> photon@0.1.0 start /Users/ggamel/Desktop/photon
> electron example-components


npm ERR! Darwin 15.0.0
npm ERR! argv "/usr/local/Cellar/node/4.1.1/bin/node" "/usr/local/bin/npm" "start"
npm ERR! node v4.1.1
npm ERR! npm  v3.3.5
npm ERR! code ELIFECYCLE
npm ERR! photon@0.1.0 start: `electron example-components`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the photon@0.1.0 start script 'electron example-components'.
npm ERR! This is most likely a problem with the photon package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     electron example-components
npm ERR! You can get their info via:
npm ERR!     npm owner ls photon
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/ggamel/Downloads/photon/npm-debug.log
```

Ditto on the dialog box.

**Comment by ggamel** [comment]
Submitted PR that fixes the Photon build issue. See: #11.

In the meantime, you can just adjust that on your own cloned repo and it should work/build, @SteveEdson. :+1:

**Comment by connors** [comment]
:+1: Fixed in #11.

**Comment by SteveEdson** [comment]
Awesome, thanks.
